client/src/index.tsx

```diff
@@ -1,11 +1,26 @@
 import React from 'react'
 import ReactDOM from 'react-dom/client'
-import App from './App'
 import './index.css'
+import Landing from './pages/Landing/Landing'
+import Presentations from './pages/Presentation'
+
+import { createBrowserRouter, RouterProvider } from 'react-router-dom'
+
+const router = createBrowserRouter([
```

Comment:
restore App and move routing logic to the app

Reply:
Agreed!